task_sdk/tests/execution_time/test_task_runner.py

```diff
@@ -318,3 +318,83 @@ def __init__(self, *args, **kwargs):
             msg=SetRenderedFields(rendered_fields=expected_rendered_fields),
             log=mock.ANY,
         )
+
+
+class TestRuntimeTaskInstance:
+    def test_get_context_without_ti_context_from_server(self, mocked_parse):
+        """Test get_template_context without ti_context_from_server."""
+        from airflow.providers.standard.operators.python import PythonOperator
+
+        task = PythonOperator(
+            task_id="hello",
+            python_callable=lambda: print("hello"),
+        )
```

Comment:
This could be BaseOperator and still work I think? If so best to avoid tests depending on code outside this project

Reply:
Yeah good idea.